src/group.h

```diff
@@ -65,6 +65,7 @@ struct GroupStatistics {
 
 enum GroupFlags : uint8 {
 	GF_REPLACE_PROTECTION, ///< If set to true, the global autoreplace has no effect on the group
+	GF_END,
```

Comment:
Put in a comment that `CmdSetGroupFlag` is currently limited to 3 bits?

Reply:
Hmm, yeah, this is wrong as I used GroupFlags as the storage type but the values within the enum are used to set bits. Perhaps it should be:

`enum GroupFlag { ... }`
`typedef uint8 GroupFlags`

or something.